packages/web-forms/src/components/controls/NoteControl.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+import type { NoteNode } from '@getodk/xforms-engine';
+import ControlLabel from '../ControlLabel.vue';
+import ControlHint from '../ControlHint.vue';
+
+defineProps<{question: NoteNode}>();
+</script>
+
+<template>
+	<div class="note-control">
+		<ControlLabel :question="question" />
+		<ControlHint :question="question" />
+	
+		<div v-if="question.currentState.value" class="note-value">
+			{{ question.currentState.value }}
+		</div>
```

Comment:
No, I don't have a design for this. I had paired with Aly to confirm if that looks okay.

@alyblenkin can you please confirm that this is how we want to show values (if there is) of "Notes"?

![image](https://github.com/user-attachments/assets/d598eb80-0b41-4853-8a65-7a9dac1f332e)

Reply:
👍 No need for further confirmation! I just wanted to make sure we've given it some intentional thought, and have something concrete to reference when it inevitably comes back up.

Reply:
We did pair on this one because it was faster than me mocking it up. As always, we can iterate on the designs! Feel free to add to the design tweaks issue if there is anything that could use some tightening up. I think this is a great area for user feedback and to test it with some real forms.